master/internal/task/idle_timeout.go

```diff
@@ -75,6 +77,12 @@ func (p *IdleTimeoutWatcher) ReceiveMsg(ctx *actor.Context) error {
 			}
 		}
 
+		if p.Debug {
+			ctx.Log().Infof("IdleTimeoutWatcherTick: lastActivity=%s timeout=%s",
+				lastActivity.Format(time.RFC3339), lastActivity.Add(p.Timeout).Format(time.RFC3339),
```

Comment:
not critical, just more FYI it exists
```suggestion
			ctx.Log().WithFields(
				"lastActivity", lastActivity.Format(time.RFC3339),
				"timeout", lastActivity.Add(p.Timeout).Format(time.RFC3339),
			).Infof("idle timeout watcher ticked"),
```

Reply:
sorry i didnt actually make sure the suggestion was right (has extra paren)